include/rocksdb/utilities/transaction.h

```diff
@@ -205,6 +205,31 @@ class Transaction {
                                        const std::vector<Slice>& keys,
                                        std::vector<std::string>* values) = 0;
 
+  // Batched version of MultiGet - see DBImpl::MultiGet(). Sub-classes are
+  // expected to override this with an implementation that calls
+  // DBImpl::MultiGet()
+  virtual void MultiGet(const ReadOptions& options,
+                        ColumnFamilyHandle* column_family,
+                        const size_t num_keys, const Slice* keys,
+                        PinnableSlice* values, Status* statuses,
+                        const bool /*sorted_input*/ = false) {
+    std::vector<Status> status;
+    std::vector<std::string> vals;
+    std::vector<ColumnFamilyHandle*> column_families(num_keys, column_family);
+    std::vector<Slice> user_keys;
```

Comment:
Is it a function we plan to use?
There are too many memory allocation here.
What's the problem of directly passing the user request in?

Reply:
I just wanted to provide a default implementation rather than make it abstract, for compatibility reasons. Returning ```Status::NotSupported()``` is also an option. ```TransactionBaseImpl``` overrider it anyway.

Reply:
Can the default one just call Get() individually?